README.md

```diff
@@ -15,30 +21,94 @@ For macOS we provide installers [on GitHub](https://github.com/CanonicalLtd/mult
 
 # Usage
 
-## Launch an instance
+## Find available images
+```
+$ multipass find
+Image  Aliases         Version   Description
+core                   20180419  Ubuntu Core 16
+14.04  t, trusty       20181203  Ubuntu 14.04 LTS
+16.04  x, xenial       20181207  Ubuntu 16.04 LTS
+18.04  b, bionic, lts  20181207  Ubuntu 16.04 LTS
```

Comment:
`18.04  b, bionic, lts  20181207  Ubuntu 16.04 LTS` should be changed to `18.04  b, bionic, lts  20181207  Ubuntu 18.04 LTS`